apps/OpenSpace/ext/launcher/src/sgctedit/windowcontrol.cpp

```diff
@@ -816,14 +816,22 @@ void WindowControl::onFovLockClicked() {
 }
 
 void WindowControl::updatePlanarLockedFov() {
-    const float aspectRatio = _windowDimensions.width() / _windowDimensions.height();
-    const float ratio = aspectRatio / IdealAspectRatio;
-    if (ratio >= 1.f) {
-        _planar.fovH->setValue(std::min(DefaultFovH * ratio, 180.f));
-        _planar.fovV->setValue(DefaultFovV);
+    bool landscapeOrientation = (_windowDimensions.width() >= _windowDimensions.height());
+    float aspectRatio;
+    if (landscapeOrientation) {
+        aspectRatio = _windowDimensions.width() / _windowDimensions.height();
     }
     else {
-        _planar.fovH->setValue(DefaultFovH);
-        _planar.fovV->setValue(std::min(DefaultFovV / ratio, 180.f));
+        aspectRatio = _windowDimensions.height() / _windowDimensions.width();
     }
+
+    const float lockedFov = DefaultFovV;
```

Comment:
Do we need the `lockedFov` variable?  Why not use the `DefaultFovV` in the terniary below directly?

Reply:
Reason is for code readability. The `DefaultFovV` is used elsewhere mainly for initializing values, for the assumed orientation (which is landscape). But if a portrait orientation is selected, then this value will be used as the horizontal fov.
Here, I chose to call the angle that we don't change `lockedFov`, and the one that we change in order to make the view look correct is `adjustedFov`.

Reply:
I get the reasoning. The name "DeafultFovV" already implies to me that this is a value that is not changed, however :)

Using it directly in the ternary would IMO also make the ternary a bit clearer to me as well, since it then is clearer that this FOV is the default **vertical** fov that is used. Hope you get what I mean

Reply:
The readability concern was with the bottom 2 lines, where the `_planar.fovH` might be set to `DefaultFovV` depending on the orientation.
However, to address the concerns with this, I got rid of the added `lockedFov` variable and renamed `DefaultFovV` & `DefaultFovH` to `DefaultFovShortEdge` and `DefaultFovLongEdge`. These names make sense with either orientation.